dev/com.ibm.ws.cdi.internal/src/com/ibm/ws/cdi/internal/interfaces/ExtensionArchiveProvider.java

```diff
@@ -29,6 +33,7 @@ public interface ExtensionArchiveProvider {
      * @return a collection of additional extension archives to add
      * @throws CDIException if an unexpected exception is encountered
      */
-    Collection<ExtensionArchive> getArchives(CDIRuntime cdiRuntime, WebSphereCDIDeployment deployment) throws CDIException;
-
+    default Collection<ExtensionArchive> getArchives(CDIRuntime cdiRuntime, WebSphereCDIDeployment deployment) throws CDIException {
+        return Collections.emptySet();
+    }
```

Comment:
This change is no longer needed since it's the only method on the interface.

The rest of the changes to this file (copyright date, imports) should also be reverted.